app/src/main/kotlin/no/nav/k9punsj/integrasjoner/punsjbollen/PunsjbolleService.kt

```diff
@@ -13,13 +14,15 @@ interface PunsjbolleService {
         journalpostId: String?,
         periode: PeriodeDto?,
         fagsakYtelseType: FagsakYtelseType,
+        correlationId: String = UUID.randomUUID().toString()
```

Comment:
Kan du legge til correlationId i coroutineScope eller MDC og hente det når det er bruk for det istedenfor å sende den som parameter i alle de funksjonene her?

Reply:
Tror ikke det virker for utgående kall, jag fjernet jo den her funktionaliteten for en tid sen men begynner o bli irriterande att feilsøke uten correlationId.
Så tenkte vi kan ta tilbake den igen och hellre la den ligge tills vi får tid till att håndtere innsending och ruting i punsj och kan fjerne punsjbollen helt?

Reply:
Ok 👍